public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -156,11 +159,12 @@ export const NavToolbarActions = React.memo<Props>(({ dashboard }) => {
         <Button
           onClick={() => {
             DashboardInteractions.toolbarSaveClick();
-            dashboard.onSave();
+            dashboard.openSaveDrawer({});
           }}
           tooltip="Save changes"
           key="save"
-          disabled={!isDirty}
+          size="sm"
+          variant={isDirty ? 'primary' : 'secondary'}
```

Comment:
How about disabling the button instead of making it secondary when there are no changes to save?

Reply:
@dprokop we don't treat variable and time range changes as "dirty" so would not be able to save those